src/Aspire.Dashboard/Components/Controls/PropertyGrid.razor

```diff
@@ -1,26 +1,29 @@
-﻿@using Aspire.Dashboard.Resources
+﻿@using Aspire.Dashboard.Components.Controls.Grid
+@using Aspire.Dashboard.Resources
 @typeparam TItem
 @inject IStringLocalizer<ControlsStrings> Loc
 
-<FluentDataGrid Items="@Items"
+<FluentDataGrid ResizeLabel="@AspireFluentDataGridHeaderCell.GetResizeLabel(Loc)"
```

Comment:
Hmm, I think there might be the opportunity to reduce the amount of change by having a custom FluentDataGrid type instead of custom columns. The custom grid type could loop through the columns and set `HeaderCellItemTemplate = AspireFluentDataGridHeaderCell.RenderHeaderContent(Grid);` on all the columns.

However, that might create problems of its own if there are situations where that is wrong behavior. Custom columns that set the header is probably ok for now. It's verbose, but it gives us control.